docs/technologies.md

```diff
@@ -79,6 +79,36 @@ For this boilerplate C++ backend template that is:
 
 ## Web Framework
 
+At the time of research, the following versions were used:
+- **Drogon:** 1.9.10
+- **userver:** 2.12
+- **Oat++:** 1.3.0
+
```

Comment:
Add here `Key factors for decision-making:` and `Comparative Table of Popular Solutions`